portal_pages/models.py

```diff
@@ -208,7 +208,7 @@ class HomePage(Page):
 
 class HomePageHeroImageItem(Orderable, models.Model):
     home_page = ParentalKey(HomePage, related_name='hero_items')
-    blurb = RichTextField()
+    blurb = models.CharField(max_length=255)
```

Comment:
Yes that's true! The model change will require us to edit these elements right after we push this out, but it's a change for the better. That field shouldn't be rich text really.